harness/tests/tensorboard/test_metric_writers.py

```diff
@@ -39,26 +39,24 @@ def test_batch_metric_writer(mock_get_base_path: mock.MagicMock, tmp_path: pathl
 
     validation_period = 2
 
-    for i in range(10):
-        step = i + 1
-        batch_writer.on_train_step_end(steps_completed=i, metrics={"x": step})
-        if i % validation_period == 0:
-            batch_writer.on_validation_step_end(steps_completed=i, metrics={"x": step})
-
-    # Sleep for 1 second to make subsequent metrics write to a new file.
-    time.sleep(1)
+    num_steps = 20
```

Comment:
maybe, uh, 6 will be enough? it'll also fit a few validation periods and it'll make the test run 3x as fast?